docs/tutorial/volumes/storageclass.md

```diff
@@ -0,0 +1,76 @@
+### Topology Aware StorageClasses
+
+There can be a use case where we have a certain set of nodes that we want to
+dedicate as storage Nodes and wants to deploy the CSI Node driver on that nodes
+and later we want a particular type of application to use those Nodes.
+
+We can create a storage class with `allowedTopologies` and mention all the storage
+nodes :
+
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: cstor-sparse-auto
+provisioner: cstor.csi.openebs.io
+allowVolumeExpansion: true
+volumeBindingMode: WaitForFirstConsumer
+parameters:
+  replicaCount: "3"
+  cstorPoolCluster: "cspc-ssd"
+  cas-type: "cstor"
+allowedTopologies:
+- matchLabelExpressions:
+  - key: "topology.cstor.openebs.io/nodeName"
+    values:
+      - node-1
+      - node-2
+      - node-3
+```
+
+The CStor CSI driver will create the Volume in the CStor Pool instance present on
+this list of nodes. `volumeBindingMode: WaitForFirstConsumer` to let the k8s select
```

Comment:
nit: *We can use*  `volumeBindingMode: WaitForFirstConsumer` to let the k8s select...